hugo/layouts/partials/card.html

```diff
@@ -0,0 +1,11 @@
+<article class="cards__element cards__element_category">
```

Comment:
I tried to follow this [BEM docummentation](https://en.bem.info/methodology/quick-start/#modifier)
>_The modifier name is separated from the block or element name by a single underscore (_)_
What means `--`?